src/api/attendance.ts

```diff
@@ -125,23 +126,21 @@ const getAttendanceForMember = async ({ season, memberId }) => {
  * Fetch Attendance for all members within the season (so far)
  * Seaason is the year the game comes out (in January)
  */
-const getAttendanceForAllMembers = async (season: string) => {
+const getAttendanceStatsForAllMembers = async (season: string) => {
     const { startDate, endDate } = getStartEndOfSeason(season)
     const theEnd = isBefore(getToday(), toDate(endDate)) ? toYMD(getToday()) : endDate
```

Comment:
Was this needed? Was it counting in the future? Was wondering if we have other places the getStartEndOfSeason might be wrong.

Reply:
I did not test that function, but I will look into it – I was wondering the same thing myself.